docs/ContribOperators.md

```diff
@@ -20,6 +20,7 @@ Do not modify directly.*
   * <a href="#com.microsoft.DecoderAttention">com.microsoft.DecoderAttention</a>
   * <a href="#com.microsoft.DequantizeBFP">com.microsoft.DequantizeBFP</a>
   * <a href="#com.microsoft.DequantizeLinear">com.microsoft.DequantizeLinear</a>
+  * <a href="#com.microsoft.DequantizeWithOrder">com.microsoft.DequantizeWithOrder</a>
```

Comment:
is this expected?

Reply:
These ops were added by this pr https://github.com/microsoft/onnxruntime/pull/12949, which didn't update the docs.